tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -1996,6 +1996,48 @@ public function testUpsertDocument(array $data): void
             ],
         ]);
         $this->assertEquals(2, $documents['body']['total']);
+
+        // test without passing permissions
+        $document = $this->client->call(Client::METHOD_PUT, '/databases/' . $databaseId . '/collections/' . $data['moviesId'] . '/documents/' . $documentId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'data' => [
+                'title' => 'Thor: Ragnarok',
+                'releaseYear' => 2000
+            ]
+        ]);
+
+        $this->assertEquals(200, $document['headers']['status-code']);
+        $this->assertEquals('Thor: Ragnarok', $document['body']['title']);
+
+        $document = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $data['moviesId'] . '/documents/' . $documentId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        if (isset($document['body']['$permissions'])) {
+            $this->assertCount(3, $document['body']['$permissions']);
+            $this->assertContains(Permission::read(Role::users()), $document['body']['$permissions']);
+            $this->assertContains(Permission::update(Role::users()), $document['body']['$permissions']);
+            $this->assertContains(Permission::delete(Role::users()), $document['body']['$permissions']);
+        }
```

Comment:
Please fix the PR so the tests pass.

Reply:
We still want a test to make sure upserting w/o permissions works. We just don't need to check the permissions right now.